sprint06/sprint-mission-6.sql

```diff
@@ -0,0 +1,203 @@
+            classic_dlx_m | classic_dlx | M    |    16 |           1181
+            spicy_ital_l  | spicy_ital  | L    | 20.75 |           1109
+            hawaiian_s    | hawaiian    | S    |  10.5 |           1020
+            southw_ckn_l  | southw_ckn  | L    | 20.75 |           1016
+            bbq_ckn_l     | bbq_ckn     | L    | 20.75 |            992
+            bbq_ckn_m     | bbq_ckn     | M    | 16.75 |            956
+        ```
+        */
+
+
+    -- 2. `orders` 테이블에서 2025년 3월의 일별 주문 수량을 `total_orders`라는 이름으로, 일별 총 주문 금액을 `total_amount`라는 이름으로 포함해서 조회하세요.
+    - SELECT orders.date,
+             COUNT(DISTINCT orders.id) AS total_orders,
+             SUM(order_details.quantity * pizzas.price) AS total_amount
+      FROM orders
+      JOIN order_details ON orders.id = order_details.order_id
+      JOIN pizzas ON order_details.pizza_id = pizzas.id
+      WHERE orders.date >= '2025-03-01' AND orders.date < '2025-04-01'
+      GROUP BY orders.date
+      ORDER BY orders.date;
+
+        /*
+        출력 예시:
+        
+        ```sql
+        2025-03-01 |           99 | 1598.5500011444092
+        2025-03-02 |          138 |  2379.050001144409
+        2025-03-03 |          133 | 2287.8999996185303
+        2025-03-04 |          144 |  2444.300001144409
+        2025-03-05 |          140 |  2350.650005340576
+        ```
+        */
+
+    -- 3. `order`의 `id`가 78에 해당하는 주문 내역들을 조회합니다. 주문 내역에서 각각 주문한 피자의 이름을 `pizza_name`, 피자의 크기를 `pizza_size`, 피자 가격을 `pizza_price`, 수량을 `quantity`, 각 주문 내역의 총 금액을 `total_amount` 라는 이름으로 조회해 주세요.
+    - SELECT pizza_types.name AS pizza_name,
+             pizzas.size AS pizza_size,
+             pizzas.price AS pizza_price,
+             order_details.quantity,
+             order_details.quantity * pizzas.price AS total_amount
+      FROM order_details
+      JOIN pizzas ON order_details.pizza_id = pizzas.id
+      JOIN pizza_types ON pizzas.type_id = pizza_types.id
+      WHERE order_details.order_id = 78;
+
+        /*
+        출력 예시:
+        
+        ```sql
+        The Thai Chicken Pizza      | S          |       12.75 |        1 |              12.75
+        The Big Meat Pizza          | S          |          12 |        1 |                 12
+        The Classic Deluxe Pizza    | S          |          12 |        1 |                 12
+        The Italian Capocollo Pizza | M          |          16 |        1 |                 16
+        The Spicy Italian Pizza     | L          |       20.75 |        3 |              62.25
+        The Four Cheese Pizza       | L          |       17.95 |        1 | 17.950000762939453
+        ```
+        */
+  
+    -- 4. `order_details`와 `pizzas` 테이블을 JOIN해서 피자 크기별(S, M, L) 총 수익을 계산하고, 크기별 수익을 출력하세요.
+    - SELECT pizzas.size,
+             SUM(order_details.quantity * pizzas.price) AS total_revenue
+      FROM order_details
+      JOIN pizzas ON order_details.pizza_id = pizzas.id
+      GROUP BY pizzas.size
+      ORDER BY pizzas.size;
```

Comment:
출력 순서에 대한 가이드는 없었지만, 예시에 따르면 수익별 DESC 되어있어서 아래와 같이 적어볼 수 있을 것 같습니다.
```suggestion
      ORDER BY total_revenue DESC;
```